drivers/access_logs/app/models/access_logs/report.rb

```diff
@@ -24,7 +25,7 @@ def filter # rubocop:disable Lint/DuplicateMethods
   def data
     {
       'Warehouse' => ActivityLog.to_a(user_id: filter.user_id, range: filter.range),
-      'CAS' => (Cas::ActivityLog.to_a(range: filter.range) unless filter.user_id.present?),
```

Comment:
Ah!  I had missed this.

Reply:
yeah I had it so the CAS tab only shows up if you selected "All Users." But that's definitely not clear to the user what's going to happen. this is better